src/Maui.Plugins.PageResolver.SourceGenerators/AutoDependencies.cs

```diff
@@ -191,11 +186,6 @@ private void InitialiseDependencies(IEnumerable<ITypeSymbol> types)
                 { "ExplicitTransients", new HashSet<ITypeSymbol>(comparer) }
             };
 
-            _namespaces = new HashSet<string>
-            {
-                "using Maui.Plugins.PageResolver;"
```

Comment:
Do we not need this now? I guess not.

Reply:
No, moved the using it to the very top of the SG class string